service/api_logic/sports_logic.py

```diff
@@ -14,4 +13,32 @@ def get_all_sports():
             "sport_id": sport.sport_id,
             "sport_name": sport.sport_name,
             "sport_img": sport.sport_img
-        } for sport in sports], 200
\ No newline at end of file
+        } for sport in sports], 200
+
+
+@handle_exceptions
+def get_all_leagues_by_sport(session, sport_name, page, per_page):
+    if sport_name is None or "Unknown":
+        return {"error": "Not param sport_name"}
```

Comment:
Hardcoded dictionary.

We should either create a model/class to keep it or, again, use a custom exception.

Whatever the choice - do NOT hardcode strings